README.md

```diff
@@ -20,463 +21,441 @@ This project now supports Swift 2, if you still require Swift 1.2 you can use th
 You can download Xcode 6.4 and install it outside of Applications, if you have already upgraded to Xcode 7.x
 http://developer.apple.com/devcenter/download.action?path=/Developer_Tools/Xcode_6.4/Xcode_6.4.dmg
 
+for IOS 9.0 add Transport Security into info.plist
+
+```xml
+<key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key>
+        <true/>
+    </dict>
+```
+
 ## Installation
 
 ExpSwift is available through [CocoaPods](http://cocoapods.org). To install
-it, add the following line to your Podfile:
+the development branch, add the following line to your Podfile:
 
 ```ruby
 use_frameworks!
 
 pod "ExpSwift", :git => 'https://github.com/ScalaInc/exp-ios-sdk.git', :branch => 'develop'
 ```
 
-### ExpSwift.start(host,uuid,secret)
-Init exp connection for device with Host,Uuid,secret. 
-```swift
-import ExpSwift
+Or to use a specific release:
```

Comment:
the correct order should be firs no tag, second tag release that is the order that I have on the readme